apps/meteor/client/views/admin/settings/Setting.tsx

```diff
@@ -119,6 +119,21 @@ function Setting({ className = undefined, settingId, sectionChanged }: SettingPr
 		[shouldDisableEnterprise, t],
 	);
 
+	const label = useMemo(() => {
+		if (!shouldDisableEnterprise) {
+			return labelText;
+		}
+
+		return (
+			<>
+				<Box is='span' mie={4}>
+					{labelText}
+				</Box>
+				<Tag variant='primary'>{t('Enterprise')}</Tag>
```

Comment:
Checking the design the color should be purple, which can be accomplished with the variant `featured`. 

About the translation key, to maintain readbility and consistence, I'd suggest to use the proper key `Premium`

```suggestion
				<Tag variant='featured'>{t('Premium')}</Tag>
```